vita3k/gui/src/settings_dialog.cpp

```diff
@@ -489,6 +512,10 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Checkbox("Enable anti-aliasing (FXAA)", &config.enable_fxaa);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Anti-aliasing is a technique for smoothing out jagged edges.\n FXAA comes at almost no performance cost but makes games look slightly blurry.");
+        ImGui::SameLine();
+        ImGui::Checkbox("V-Sync", &config.v_sync);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Unchecking the box may fix the speed issue on some games.\nIt is recommended to keep it checked.");
```

Comment:
...can improve the speed in some games.
It is recommended to keep it checked to avoid tearing.

Reply:
ah yes, but for first line, i not imporove, but fix speed issue, no want said same things